contacts/src/App.css

```diff
@@ -0,0 +1,86 @@
+/* Existing styles for App logo animation */
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+/* Override the styles for the contacts app */
+.App {
+  text-align: left; /* Align text to the left for the contact app */
+  background-color: #151b1d; /* Set the background color for the contacts app */
+  color: #ecf2f4; /* Set font color for the contacts app */
+  font-family: "Space Grotesk", sans-serif; /* Set the main font for the app */
+}
+
+/* Contact app specific styles */
+
+/* Header styles */
+.header {
+  padding: 20px;
+  text-align: center; /* Center align the header content */
+  font-weight: 700; /* Use bold for the header title */
+}
+
+/* Contact Form styles */
+/* You would add styles for your form inputs, labels, and buttons here */
+
+/* Contact List styles */
+.contact-list {
+  padding: 20px; /* Add padding around the contact list */
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+/* Contact Card styles */
+.contact-card {
+  border: 1px solid #5a7d7c;
+  margin-bottom: 10px;
+  padding: 10px;
+  text-align: left; /* Align the text to the left for contact card */
+}
+
+/* Heart Icon styles */
+.heart-icon {
+  cursor: pointer;
+  color: #e36940; /* Set color for the heart icon */
+  transition: transform 0.3s ease; /* Transition for hover effect */
+}
+
+.heart-icon:hover {
+  transform: scale(1.1); /* Grow the icon slightly on hover */
+}
+
+/* Media query for desktop design */
+@media (min-width: 768px) {
+  /* Adjust the flexbox layout for desktop */
+  .home {
+    display: flex;
+    justify-content: space-between;
```

Comment:
Nice css setup, but you didn't assign the className "home" in any of your components. I assume you wanted it in the top div of your Home component. If you added the property flex-direction: row here, you would get a side by side display in desktop view.